wave-app/src/data.py

```diff
@@ -16,12 +17,13 @@ async def data(q:Q):
 
     # Display the head of the dataframe as a ui markdown table.
     df = q.app.datasets[val]
-    df_head = df.head(10)
+    df_head = df.sample(6)
     df_table = await make_markdown_table(
         fields=df_head.columns.tolist(),
         rows=df_head.values.tolist()
     )
     q.page['df'] = ui.form_card(box=ui.box('data'), items=[
+        ui.text(FIRMS_INFO),
```

Comment:
Could you please add this **after** the `datasets` combobox / `describe` button in a separate card (or in the `data` card only) so that the info looks like it belongs to and loads after the dataset is Selected?